.github/workflows/check-standard.yaml

```
@@ -0,0 +1,46 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/master/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+on:
+  push:
+    branches: [main, master]
```

```suggestion
    branches: [main, master, develop]
```
It should be fine adding the checks for develop branch too.